### PR TITLE
Update setup instructions + remove unused env vars

### PR DESCRIPTION
**Pull request by Lontronix**
The environment variables IP and PORT did not appear to be used so I removed them from config.py and the template envs.py in README.md

I also updated the template envs.py to have the minimum required environment variables to get the program up and running locally.

I wasn't able to get the program up and running locally using the previous instructions without a bunch of debugging.

### CODE REVIEW

File: README.md
Comment (MTFT-Games) [inline README.md]: This link has no opening bracket.

**Review by MTFT-Games — CHANGES_REQUESTED** [review]
Other than broken link I will defer to someone that knows how flask works cc @esoccoli perhaps

**Review by pikachu0542 — COMMENTED** [review]
I tested it with these instructions, and it worked. Once the link is fixed ill give it an approval

File: README.md
Comment (Lontronix) [reply README.md]: Good catch! Just fix it

**Review by pikachu0542 — APPROVED** [review]
lgtm

**Review by MTFT-Games — APPROVED** [review]

**Pull request merged**
